src/Tracking/Repositories/TelemetryAccessDetails.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Give\Tracking\Repositories;
+
+/**
+ * Class TelemetryAccessDetails
+ * @package Give\Tracking\Repositories
+ *
+ * @since 2.10.0
+ */
+class TelemetryAccessDetails {
+	/**
+	 * Get option key for telemetry access token.
+	 *
+	 * @since 2.10.0
+	 * @return string
+	 */
+	public function getAccessTokenOptionKey() {
+		return 'give_telemetry_server_access_token';
```

Comment:
Make this a constant, too.

Reply:
@JasonTheAdams I made changes. I did that because I saw a code sample in core: https://github.com/impress-org/givewp/blob/719e6079ba1c49066f863b1515159e539bf3652a/src/PaymentGateways/PayPalCommerce/Repositories/MerchantDetails.php#L175

Reply:
Yes, but notice that the line has an injected instance variable, `$this->mode`; it's not actually a constant.

Reply:
I got your point.